helpers/icons-at-least.js

```diff
@@ -0,0 +1,34 @@
+/**
+ * @license
+ * Copyright 2016 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+'use strict';
+
+/**
+ * @param  {Number} sizePx
+ * @param  {any} manifest
```

Comment:
oh, actually you want `{!ManifestNode<(!Manifest|undefined)>}` (the type of `artifacts.manifest`). *But*, since you already know at the call sites that there's a `manifest.value`, you could pass in `artifacts.manifest.value` instead and type it as a simple `{!Manifest}`